### DISCUSSION TITLE
"Answered" seems a misleading moniker, since it does not convey "resolved" or "accepted"

### DISCUSSION THREAD

**Discussion by hepcat72** [community/community#3143] [Discussions]
I suppose I could be conflating stack's concept of an (accepted) "answer" with github discussion's concept of "answered".

My first usage of github discussions, was to start a discussion, which I thought would be a back-and-forth sort of informal/casual interaction.  I didn't initially think of it as asking a question to be "answered".  I didn't even realize at first, that github discussions contained a mechanism for "answering" (Upon my first visit, I assumed it would be more like a generic topic-based "discussion" forum).  After receiving the first response to my post, and seeing it as having been marked (by the responder?) as "answered" (and seeing that the answer was "wrong"), I was a bit surprised.

Clearly, I had misinterpreted what the github discussion feature was for and I re-interpreted it to be closer to a stack-exchange sort of interaction, but given that the "answer" was not helpful in the sense that it was not the end of the discussion, and that I wasn't involved in determining whether my question(s) were "answered", I noted that it's not quite the same paradigm as stack either.

I'm certain that there is a well reasoned meaning behind how the answering mechanism was implemented, and I see that I can un-mark the selected answer, so I again re-assessed what information "answered" is intended to convey.  Here are the possible meanings I came up with:

1. The question has been "responded" to by whomever owns or contributes to a repo and is intended as a sort of ticket tracking feature
2. The question has been conclusively and completely answered for the questioner (assuming the question was well crafted and clear)
3. It's a proposal asking if the question has been answered and is useful to the questioner
4. It's an indicator of a response being accepted/useful to the questioner

My guess is that it's some combination of all 4, but depending on perspective, seems to be easily misinterpreted.  If this is true, I would think that there might be a better word than "answered", though I would think there should be more than one word/state that an "answer" can be in?  I'm also unclear on how/who/when a response is marked as "answered".

I'm on-board with a new paradigm that is not a simple discussion forum and is not a stack-exchange site.  I'm just saying my first impression is that it's not clear to me how this works and that perhaps an alternate word choice other than "answered" for categorizing a response might help - or maybe upon a user first visiting any github discussion page, a user is given a quick intro on the functionality, or what the intended use of it is...

**Comment by evi-liu** [comment]
Thanks for taking the time to write such detailed feedback. I'm curious if others have the same perception of `answered` and if it's leading to confusion.

**Reply by kovitikus** [reply]
It's not that confusing to me, because I understood that only certain categories of the Discussions section are marked as a Question and Answer format. If I had not known that before participating, I might have been confused.

@hepcat72 Not every discussion has the ability to be marked as an answer. The person in control of the github must specify that a category is of a question/answer format. It seems you were just unlucky to have stumbled into the Q/A section as your first introduction to the Discussions feature, or the place where you posted had poorly designated categories.

For me Answered means that the question has been resolved to the author's satisfaction. There may be further discussion and perhaps a new and better answer can be selected in the future, but for now it's resolved.

As pointed out in #3190 it does seem that allowing someone else other than the author of the thread to select an answer can lead to some very confusing results.

What if you have multiple maintainers on a GitHub and they decide to keep changing around the post marked as answered? It's a silly concept and it reeks a bit of, "I know better than you if this answers your question." That simply isn't always the case ~~and I'm coming to the conclusion that only the author of the post should be able to make that decision.~~ (Changed my mind as per discussion in #3190)

**Reply by hepcat72** [reply]
I can add the following... I am actively wondering if any response is auto-selected as an answer, because I keep deselecting the "selected answer" and when I receive a reply from one of the repo contributors, their response is selected as the "answer" even if that response is clearly not an answer.

I further observed (my own behavior) that I stopped deselecting the "selected answer" because it seemed fruitless, i.e. my deselection never stuck.  I believe that this lessens the value of the "selected answer" feature.  If, like me, users are not diligent with deselecting non-answers, then people coming to the discussion may occasionally see "answers" that are not "answers" and eventually lose trust in posts marked as "answers".  Then you may start getting people adding comments like "Did this actually resolve your question - I'm having the same issue and this did not work for me".

I will say that the name of the particular forum I posted in under the repo was "Q&A", but that's under the "Discussions" tab, which makes it slightly ambiguous.  The perspective I have is that I did have questions, but they were detail questions to help me investigate a larger question that was in the heading of my post.  So Q&A seemed like the right place.  It's just that the over-arching question is not really answered.  And the current "selected" answer is just a suggestion that advises me how to turn on a debugging mode with a suggested log level.

**Reply by MgSam** [reply]
The terminology definitely does not make sense. A "discussion" cannot be "answered". A question can be answered. The feature should either be called "Questions" or the terminology of "answered" should be changed. Maybe "best response"?

**Comment by TinCanTech** [comment]
I also have a **Wrong** "Answer":
https://github.com/github/feedback/discussions/3148#discussioncomment-294733

**Comment by ocornut** [comment]
I admit it really rubs me off that:
- Discussions cannot be marked as "Resolved" without enabling the "Question / Answer" feature.
- The "Question / Answer" feature put an emphasis on a single "right answer" which I believe is tremendously harmful to healthy discussion. In the majority of threads, most posts include components of right and wrong. 
- The "Question / Answer" enable those gamification features which incentive people to answer to get points which IHMO is a recipe for low signal to noise ratio. 

Ideally I would imagine it would be nice if:
- Multiple posts could be marked as "most useful" by author or project maintainers. Perhaps the voting can already convey that.
- Discussions can be marked as Resolved, aka, the author problem was resolved and maintainers/helpers don't need to particularly act on this. This without enforcing a "Single right answer" paradigm.

Also note that allowing posters and maintainers to Labels discussions #19 (which is my most wanted feature at this point) or even individual posts, with different sets of labels for "asker" and "mainteners" would somehow allow implementing varieties of those ideas.
As a question asker a small selection of labels could allow expressing this. (As a maintainer I want to categorize posts by e.g. features, which is a widely different area, but they could both be conveyed by labels)

**Reply by nagyrobi** [reply]
Fully agree.
